vizro-core/src/vizro/models/_base.py

```diff
@@ -278,14 +278,15 @@ def _to_python(
 
             >>> import vizro.models as vm
             >>> card = vm.Card(text="Hello, world!")
-            >>> print(card._to_python())
+            >>> card._to_python()
```

Comment:
isnt it nicer to use `print` to see the result?

Reply:
No, because if you actually do this on the REPL then it will show you the result anyway. Just like doing it in Jupyter. That's the most natural way to "read" a docstring example rather than including the print in the example itself.

Reply:
But it shows it without the formatting, e.g. linebreaks `\n` no?

Reply:
Ah ok yes, let me revert this then 👍